src/styles/index.scss

```diff
@@ -0,0 +1,101 @@
+/**
+ * Naming scheme for variables is `--component-name-size-property-modifier`.
+ */
+
+@import './variables.scss';
+
+:root {
+  /* Colors */
+  --color-primary: #{$ui-03};
+  --color-secondary: #{$ui-04};
+  --border-color-light: #dddddd;
+  --background-color-primary: #{$ui-background-01};
+  --background-color-secondary: #{$ui-background-02};
+  --text-color-primary: #{$black};
+
+  --gds-color-white: #{$white};
+  --gds-color-black: #{$black};
+  --gds-color-ui-background-01: #{$ui-background-01};
+  --gds-color-ui-background-02: #{$ui-background-02};
+  --gds-color-ui-01: #{$ui-01};
+  --gds-color-ui-02: #{$ui-02};
+  --gds-color-ui-03: #{$ui-03};
+  --gds-color-ui-04: #{$ui-04};
+  --gds-color-ui-05: #{$ui-05};
+  --gds-color-ui-06: #{$ui-06};
+  --gds-color-ui-07: #{$ui-07};
+
+  /* Spacing */
+  --spacing-xxxs: #{$spacing-xxxs};
+  --spacing-xxs: #{$spacing-xxs};
+  --spacing-xs: #{$spacing-xs};
+  --spacing-s: #{$spacing-s};
+  --spacing-m: #{$spacing-m};
+  --spacing-l: #{$spacing-l};
+  --spacing-xl: #{$spacing-xl};
+  --spacing-xxl: #{$spacing-xxl};
+
+  /* Body */
+  --body-font-family: Gilroy, Arial, Helvetica, sans-serif;
+  --body-background-color: var(--background-color-primary);
+  --body-text-color: var(--text-color-primary);
+
+  /* Headings */
+  --heading-font-family: Gilroy, Arial, Helvetica, sans-serif;
+
+  /* Labels */
+  --label-font-family: Gilroy, Arial, Helvetica, sans-serif;
+  --label-s-font-weight: 600;
+  --label-m-font-weight: 500;
+  --label-l-font-weight: 400;
+  --label-s-font-size: 10px;
+  --label-m-font-size: 12px;
+  --label-l-font-size: 16px;
+  --label-s-letter-spacing: 0.5px;
+  --label-m-letter-spacing: 1px;
+  --label-l-letter-spacing: 1.28px;
+
+  /* Tags */
+  --tag-color: white;
+  --tag-background-color: black;
+  --tag-border-color: transparent;
+
+  /* Buttons */
+  --button-font-family: Gilroy, Arial, Helvetica, sans-serif;
+  --button-color: white;
+  --button-background-color: $black;
+  --button-background-color-hover: $ui-02;
```

Comment:
I feel like these should use the the css variables no?

```css
--button-background-color-hover: var(--gds-color-ui-02);
```

Reply:
Yeah I think they should as you should be able to change to override the --gds-color-ui-02 from outside GDS. Right?

Reply:
OK.